src/web/assets/cp/src/js/ElementEditor.js

```diff
@@ -589,6 +611,255 @@ Craft.ElementEditor = Garnish.Base.extend(
+        value: s.id,
+      }));
+    },
+
+    _getCopyBetweenSitesForm: function ({
+      fieldHandle = null,
+      elementId = this.settings.canonicalId,
+      namespace = this.namespace,
+    } = {}) {
+      const $form = Craft.createForm();
+      $form.append(Craft.getCsrfInput());
+
+      if (fieldHandle) {
+        $('<input/>', {
+          type: 'hidden',
+          name: 'fieldHandle',
+          value: fieldHandle,
+        }).appendTo($form);
+      }
+
+      if (elementId) {
+        $('<input/>', {
+          type: 'hidden',
+          name: 'elementId',
+          value: elementId,
+        }).appendTo($form);
+      }
+
+      // Namespace for slideouts
+      if (namespace) {
+        $('<input/>', {
+          type: 'hidden',
+          name: 'namespace',
+          value: namespace,
+        }).appendTo($form);
+      }
+
+      const $fields = $('<div/>', {
+        class: 'flex flex-end flex-nowrap',
+      });
+
+      const $siteSelect = Craft.ui.createSelectField({
+        label: Craft.t('app', 'Copy from'),
+        name: 'copyFromSiteId',
+        class: ['fullwidth'],
+        options: this._getSitesForCopyFieldAction(),
+      });
+
+      // `class` above targets the select input, we need the container
+      $siteSelect.addClass('flex-grow');
+      $fields.append($siteSelect);
+
+      const $submitBtn = Craft.ui.createSubmitButton({
+        label: Craft.t('app', 'Copy'),
+        spinner: true,
+      });
+      $fields.append($submitBtn);
+
+      $form.append($fields);
+
+      this.addListener($form, 'submit', 'copyValuesFromSite');
+
+      return $form;
+    },
+
+    showElementCopyModal: function (ev) {
+      const headingId =
+        'elementCopyModalHeading-' + Math.floor(Math.random() * 1000000);
+
+      const $hudContent = $('<div/>', {
+        class: 'modal fitted copy-translation-dialogue',
+        'aria-labelledby': headingId,
+      });
+      const $body = $('<div/>', {
+        class: 'body',
+      });
+
+      $body.append(
+        `<h1 id="${headingId}" class="h2">${Craft.t(
+          'app',
+          'Copy content from site'
+        )}</h1>`
+      );
+
+      $body.append(
+        $(
+          `<p>${Craft.t(
+            'app',
+            'Only translatable <span aria-hidden="true">(<span class="t9n-indicator" data-icon="language" data-handle="title"></span>)</span> field values will be copied, and only if their content differs from the current.'
+          )}</p>`
+        )
+      );
+
+      $body.append(this._getCopyBetweenSitesForm());
+      $hudContent.append($body);
+
+      this.copyModal = new Garnish.Modal($hudContent);
+
+      this.copyModal.on('hide', function () {
+        // return focus to the actions btn
+        const actionMenu = $('#action-menu')
+          .disclosureMenu()
+          .data('disclosureMenu');
+        actionMenu.$trigger.focus();
+      });
+
+      this.addListener($('.copyBetweenSites'), 'submit', 'copyValuesFromSite');
```

Comment:
@i-just As far as I can tell, this selector won’t ever match anything. Am I missing something? (Same for line 747.)

Reply:
No, you’re not missing anything. They were used in the older version of this feature, but not anymore. I removed them.